### PR TITLE
Bug 1745630: chore(deps): update grpc dependency

### PR DESCRIPTION
**Pull request by ecordell**
Address CVEs in grpc-go

### CODE REVIEW

File: test/e2e/operator_groups_e2e_test.go [hunk 1]
```diff
@@ -523,6 +523,28 @@ func TestOperatorGroupRoleAggregation(t *testing.T) {
 		return res.Status.Allowed, nil
 	})
 	require.NoError(t, err)
+	err = wait.Poll(pollInterval, pollDuration, func() (bool, error) {
```
Comment (mfojtik) [inline test/e2e/operator_groups_e2e_test.go]: @ecordell how is this related to grpc bump?

Comment (ecordell) [reply test/e2e/operator_groups_e2e_test.go]: Thanks for catching this - must've been a weird rebase issue.

**Comment by mfojtik** [comment]
/retitle Bug 1746659: chore(deps): update grpc dependency

**Comment by eparis** [comment]
/retitle Bug 1746659: chore(deps): update grpc dependency
